src/main/java/org/lsc/plugins/connectors/james/JamesDao.java

```diff
@@ -64,10 +65,12 @@
 public class JamesDao {
```

Comment:
If I am not mistaken we do have unit tests for JamesDao...

Reply:
Sure